packages/integration-karma/test/light-dom/shadow-getter/index.spec.js

```diff
@@ -0,0 +1,14 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+
+import Test from 'x/test';
+describe('Basic Light DOM', () => {
+    it('should render properly', () => {
+        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', true);
```

Comment:
Move flag to `before` and `after` blocks.

Reply:
`before` and `after` blocks look like they run in a different stack. This is causing failures when tests are run in parallel.

Reply:
Jasmine test runs in a serial fashion as far as I know.